unittests/tests_functions.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+from unittests.functions import Fibonacci, formatted_name
+
+
+class TestFibonacci(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.function = Fibonacci()
+
+    def test_1(self):
+        self.assertEqual(self.function(1), 1)
+
+    def test_float(self):
```

Comment:
Use more descriptive test case names to help others (and your future self) understand what is being tested. Instead of test_1, use something like test_fibonacci_of_1_returns_1.